### PR TITLE
Remove useless error-check in createImageReference

### PR DESCRIPTION
**Pull request by fuweid**
Remove the useless error check during creating reference in containerd.

ping @Random-Liu

### CODE REVIEW

File: pkg/server/image_pull.go [hunk 1]
```diff
@@ -199,7 +199,7 @@ func (c *criService) createImageReference(ctx context.Context, name string, desc
 	if err == nil {
 		return nil
 	}
-	if err != nil && !errdefs.IsAlreadyExists(err) {
+	if !errdefs.IsAlreadyExists(err) {
```
Comment (mikebrow) [inline pkg/server/image_pull.go]: nit.. can we make this an else case so it's explicit that we meant it so.. :-)

Comment (fuweid) [reply pkg/server/image_pull.go]: it is ok w/o `else` here I think...both are explicit, lol

Comment (Random-Liu) [reply pkg/server/image_pull.go]: Since last `if` return directly, it is fine not to add an else. :)

Comment (mikebrow) [reply pkg/server/image_pull.go]: I suppose if one does not like if else chains a switch is more elegant. But yeah was just a nit :-)

**Pull request merged**